modules/interactiveOffersBidAdapter.js

```diff
@@ -0,0 +1,149 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+import {config} from '../src/config.js';
+import * as utils from '../src/utils.js';
+
+const BIDDER_CODE = 'interactiveOffers';
+const ENDPOINT = 'https://rtb.ioadx.com/bidRequest/?partnerId=4a3bab187a74ac4862920cca864d6eff195ff5e4';
+
```

Comment:
please add gvlid

Reply:
For what I understand, gvlid is used for the adapter alias, we are not using alias.
Do we have to have an alias so that we have an gvlid?
![Captura de ecrã 2021-03-15, às 09 07 12](https://user-images.githubusercontent.com/76956619/111130175-ce5c8a00-856e-11eb-8e55-29bbdaf6d494.png)

Reply:
ah i was under the impression that all adapters have gvlid